app/src/main/java/com/example/ar_reshare/SplashActivity.java

```diff
@@ -59,7 +59,7 @@ private void proceed() {
         timer.schedule(new TimerTask() {
             @Override
             public void run() {
-                Intent intent = new Intent(SplashActivity.this, ARActivity.class);
+                Intent intent = new Intent(SplashActivity.this, FallbackActivity.class);
```

Comment:
I think this is should be AR Activity? 😄

Reply:
Good catch, will fix now :)